app/javascript/controllers/chat_message_controller.js

```diff
@@ -0,0 +1,12 @@
+import { Controller } from "@hotwired/stimulus"
+
+/**
+ * Add mutation to move the scroll to the bottom of the chat when
+ * it is updated.
+ */
+
+export default class extends Controller {
+  connect() {
+    this.element.scrollIntoView()
```

Comment:
This is rad!

Reply:
Jeje it is good, but actually has some issues -- which it scrolls down the entire page rather than just the internal element. I'd probably use something like -- https://github.com/stimulus-use/stimulus-use/blob/main/docs/use-mutation.md to control both the scrolling down and moving the element to the left or right depending on who sent the message.